lib/components/viewers/realtime-status-label.js

```diff
@@ -0,0 +1,95 @@
+import { formatDuration } from '@opentripplanner/core-utils/lib/time'
+import React from 'react'
+import styled from 'styled-components'
+
+import { getTripStatus, REALTIME_STATUS } from '../../util/viewer'
+
+// If shown, keep the '5 min' portion of the status string on the same line.
+export const DelayText = styled.span`
+  white-space: nowrap;
```

Comment:
Almost! I will add a spacer after this element so the rendering is correct in the stop viewer.

![image](https://user-images.githubusercontent.com/56846598/103663321-f7451b00-4f3e-11eb-877d-aa6b32109535.png)